cetmix_tower_server/wizards/cx_tower_command_execute_wizard.py

```diff
@@ -49,11 +49,22 @@ class CxTowerCommandExecuteWizard(models.TransientModel):
         help="Run commands using 'sudo'",
     )
     code = fields.Text(compute="_compute_code", readonly=False, store=True)
-    any_server = fields.Boolean()
+    any_server = fields.Boolean(default=True)
     rendered_code = fields.Text(
         compute="_compute_rendered_code",
     )
     result = fields.Text()
+    is_show_servers = fields.Boolean()
```

Comment:
Do we need to have this field stored?
Just wondering if it's possible to use computed non stored field instead.

Reply:
Thanks, I'll check it and fix